package.json

```diff
@@ -57,16 +55,16 @@
   "devDependencies": {
     "@actions/core": "^1.0.0",
     "@actions/github": "^2.2.0",
-    "@oclif/dev-cli": "^1.22.2",
-    "@oclif/test": "^1.2.6",
+    "@oclif/test": "^2.1.1",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/exec": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@twilio/cli-test": "^2.1.0",
-    "aws-sdk": "^2.757.0",
-    "chai": "^4.2.0",
+    "@twilio/cli-test": "3.0.0",
+    "aws-sdk": "^2.443.0",
```

Comment:
aws-sdk is a mandatory dependency for oclif-dev, so the version is kept in [sync](https://github.com/oclif/dev-cli/blob/ef2ec22438e87fb7107b4c49d2c47584a2222817/package.json#L36) with oclif-dev.